core/iwasm/common/wasm_application.c

```diff
@@ -354,13 +364,46 @@ union ieee754_double {
     } ieee;
 };
 
+#if WASM_ENABLE_REF_TYPES != 0
+static int32
+type_size_in_double_world(uint8 type)
```

Comment:
maybe `word` in this semantic

BTW, I wonder if there is an absolute definition of `WORD`? 
If I remembered correctly, the bit width of `WORD` may varies by platforms, it should represents `the most convenient block of data for the computer to deal with`